### DISCUSSION TITLE
How to attribute checkout via storefront-api as sales channel?

### DISCUSSION THREAD

**Discussion by tannergaucher** [Shopify/storefront-api-feedback#51] [Help] [answered]
EDIT:  Removed question for now.

I believe the issue is how we are querying attribution, rather than attribution itself.

**Comment by morganmccunn** [comment, marked as answer]
Sounds good! For reference, if the checkout was created through the channel that uses the storefront access token to power it, then the sale will be attributed to you storefront api sales channel. You'll see this come up as the name of your private app in Shopify reports.
